pkgs/development/python-modules/djangorestframework-jwt/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, fetchFromGitHub
+, django
+, pyjwt
+, djangorestframework
+, buildPythonPackage
+}:
+
+buildPythonPackage rec {
+  pname = "djangorestframework-jwt";
+  version = "1.11.0";
+
+  # PyPI tarball doesn't include the test suite
+  src = fetchFromGitHub {
+    owner = "GetBlimp";
+    repo = "django-rest-framework-jwt";
+    rev = "${version}";
+    sha256 = "1ddi1vx3yvxzsjmxx37sxnbaq27gzzwymaicqcwp9plpdzgdz79s";
+  };
+
+  propagatedBuildInputs = [ pyjwt django djangorestframework ];
+
+  doCheck = false;
```

Comment:
@dotlambda I disable the `doCheck` because tests break due to the lack of the User model in an expected database. This app should be tested as part of a django project, which there currently isn't. If you want the tests to be run I can tinker about creating a django setup for this one app in the check phase.

Reply:
However if the tests won't  run, we might as well use the pypi release. what do you think?

Reply:
Yes, please use the PyPI tarball and add a comment explaining why the tests can't be run.

Reply:
:+1:  done